src/prefect/runner/submit.py

```diff
@@ -94,30 +97,56 @@ async def _submit_flow_to_runner(
         )
         response.raise_for_status()
 
-        flow_run_id = response.json()["flow_run_id"]
+        flow_run_id = uuid.UUID(response.json()["flow_run_id"])
 
-        return uuid.UUID(flow_run_id)
+        return await client.read_flow_run(flow_run_id)
 
 
 @sync_compatible
 async def submit_to_runner(
     prefect_callable: Union[Flow, Task],
-    parameters: Dict[str, Any],
-) -> uuid.UUID:
+    parameters: Optional[Dict[str, Any]] = None,
+) -> Union[FlowRun, Any]:
     """
     Run a callable in the background via the runner webserver.
 
     Args:
         prefect_callable: the callable to run (only flows are supported for now, but eventually tasks)
         parameters: the keyword arguments to pass to the callable,
     """
-
-    flow_run_id = await _submit_flow_to_runner(prefect_callable, parameters)
-
-    if inspect.isawaitable(flow_run_id):
-        return await flow_run_id
+    if not PREFECT_EXPERIMENTAL_ENABLE_EXTRA_RUNNER_ENDPOINTS.value():
+        raise ValueError(
+            "The `submit_to_runner` utility requires the `Runner` webserver to be"
+            " built with extra endpoints enabled. To enable this, set the"
+            " `PREFECT_EXPERIMENTAL_ENABLE_EXTRA_RUNNER_ENDPOINTS` setting to `True`."
+        )
+    parameters = parameters or {}
+    try:
+        flow_run = await _submit_flow_to_runner(prefect_callable, parameters)
+    except (httpx.ConnectError, httpx.HTTPStatusError) as exc:
+        if PREFECT_RUNNER_SERVER_ENABLE_BLOCKING_FAILOVER.value():
+            logger.warning(
+                "The `submit_to_runner` utility failed to connect to the `Runner`"
+                " webserver, but blocking failover is enabled. The utility will"
+                " block until the flow run completes. You can disable this behavior"
+                " by configuring the `PREFECT_RUNNER_SERVER_ENABLE_BLOCKING_FAILOVER`"
+                " setting to `False`."
+            )
+            result = prefect_callable(**parameters)
+            if inspect.isawaitable(result):
```

Comment:
I think this is the right thing to do, I just wonder if there's a way to get a flow_run_id out of this call and return that instead (or look up it's corresponding `FlowRun` object and return that).